Makefile

```
@@ -157,12 +157,17 @@ fmt: ## Check formatting of go sources.
 	@$(MAKE) go.init
 	@$(MAKE) go.fmt
 
+
+.PHONY: markdownlint
+markdownlint: ## Check formatting of documentatio n sources
+	@$(DOCKERCMD)	run --platform linux/amd64 -v .\:/workdir\:z docker.io/davidanson/markdownlint-cli2:v0.17.1  markdownlint-cli2 "Documentation/**.md" "\#Documentation/Helm-Charts/**" --config .markdownlint-cli2.cjs
```

extra space here. have you checked to see if your editor has a "render whitespace" option or the equivalent?

```suggestion
	@$(DOCKERCMD)	run --platform linux/amd64 -v .\:/workdir\:z docker.io/davidanson/markdownlint-cli2:v0.17.1 markdownlint-cli2 "Documentation/**.md" "\#Documentation/Helm-Charts/**" --config .markdownlint-cli2.cjs
```